lib/Controller/AppSettingsController.php

```diff
@@ -55,8 +55,11 @@ public function __construct($appName,
 	 * @param string $stun_server
 	 * @param string $turn_server
 	 * @param string $turn_server_secret
+	 * @param string $signaling_server
+	 * @param string $signaling_secret
+	 * @param string $signaling_skip_verify_cert
```

Comment:
order is mixed up
can you also add `$turn_server_protocols` to the docs?

Reply:
Changed